pkg/api/types.go

```diff
@@ -1544,6 +1544,16 @@ type NodeStatus struct {
 	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
 	// Set of ids/uuids to uniquely identify the node.
 	NodeInfo NodeSystemInfo `json:"nodeInfo,omitempty"`
+	// List of container images on this node
+	Images []ContainerImage `json:"images",omitempty`
+}
+
+// Describe a container image
+type ContainerImage struct {
+	// Names by which this image is known.
+	RepoTags []string `json:"repoTags"`
```

Comment:
An image can have multiple tags, so getting a single "Name" is not enough.

An alternative is to have
```
Name string
Tags []string
```
However, since we accept `gcr.io/google_containers/busybox:1.24` as a single image string in `api.Container`, this will only make finding a match more complicated. After we settle down on what our new image spec should be, we can easily convert the RepoTags to the new format.